.prettierrc

```diff
@@ -1,7 +1,24 @@
 {
-	"useTabs": true,
-	"tabWidth": 4,
-	"semi": true,
-	"singleQuote": false,
-	"printWidth": 80
+    "tabWidth": 4,
+    "useTabs": false,
```

Comment:
A bit late, but want to argue that using tabs over spaces is more accessible for users who prefer different indentation sizes.